### ISSUE TITLE
Prisma deploy 'Ask timed out on [Actor[akka://single-server/user/$m#1622560585]] after [300000 ms]'

### ISSUE THREAD

**Issue by Lotafak** [prismagraphql/prisma#2419]
# Bug Report

**Current behavior**
running following commands:
```
docker-compose build
docker-compose up
prisma deploy
```
in main's project folder result in the following console output:
```
db_1      | Version: '5.7.22'  socket: '/var/run/mysqld/mysqld.sock'  port: 3306  MySQL Community Server (GPL)
server_1  | Server is running on http://localhost:4000
prisma_1  | Initializing workers...
prisma_1  | Successfully started 1 workers.
prisma_1  | Server running on :4466
prisma_1  | Version is up to date.
prisma_1  | {"@timestamp":"2018-05-14T09:31:20.989+00:00","@version":1,"message":"{\"key\":\"request/new\",\"requestId\":\"local:management:cjh61tq0800000c10jdmdwh9h\"}","logger_name":"com.prisma.deploy.server.ManagementServer","thread_name":"single-server-akka.actor.default-dispatcher-8","level":"INFO","level_value":20000,"HOSTNAME":"8168b4fc1bb6"}
prisma_1  | [Metrics] Warning: no metrics will be recorded.
prisma_1  | {"@timestamp":"2018-05-14T09:31:22.120+00:00","@version":1,"message":"{\"key\":\"request/complete\",\"requestId\":\"local:management:cjh61tq0800000c10jdmdwh9h\",\"payload\":{\"request_duration\":\"1182\"}}","logger_name":"com.prisma.deploy.server.ManagementServer","thread_name":"single-server-akka.actor.default-dispatcher-4","level":"INFO","level_value":20000,"HOSTNAME":"8168b4fc1bb6"}
prisma_1  | {"@timestamp":"2018-05-14T09:31:22.183+00:00","@version":1,"message":"{\"key\":\"request/new\",\"requestId\":\"local:management:cjh61tqyv00010c10okzdv5uz\"}","logger_name":"com.prisma.deploy.server.ManagementServer","thread_name":"single-server-akka.actor.default-dispatcher-3","level":"INFO","level_value":20000,"HOSTNAME":"8168b4fc1bb6"}
prisma_1  | {"@timestamp":"2018-05-14T09:31:22.201+00:00","@version":1,"message":"{\"key\":\"request/complete\",\"requestId\":\"local:management:cjh61tqyv00010c10okzdv5uz\",\"payload\":{\"request_duration\":\"18\"}}","logger_name":"com.prisma.deploy.server.ManagementServer","thread_name":"single-server-akka.actor.default-dispatcher-8","level":"INFO","level_value":20000,"HOSTNAME":"8168b4fc1bb6"}
prisma_1  | {"@timestamp":"2018-05-14T09:31:22.258+00:00","@version":1,"message":"{\"key\":\"request/new\",\"requestId\":\"local:management:cjh61tr0x00020c103z6vu7il\"}","logger_name":"com.prisma.deploy.server.ManagementServer","thread_name":"single-server-akka.actor.default-dispatcher-4","level":"INFO","level_value":20000,"HOSTNAME":"8168b4fc1bb6"}
prisma_1  | {"@timestamp":"2018-05-14T09:31:23.417+00:00","@version":1,"message":"{\"key\":\"request/complete\",\"requestId\":\"local:management:cjh61tr0x00020c103z6vu7il\",\"payload\":{\"request_duration\":\"1160\"}}","logger_name":"com.prisma.deploy.server.ManagementServer","thread_name":"single-server-akka.actor.default-dispatcher-4","level":"INFO","level_value":20000,"HOSTNAME":"8168b4fc1bb6"}
prisma_1  | {"@timestamp":"2018-05-14T09:31:23.444+00:00","@version":1,"message":"{\"key\":\"request/new\",\"requestId\":\"local:management:cjh61trxw00030c10wbyen89q\"}","logger_name":"com.prisma.deploy.server.ManagementServer","thread_name":"single-server-akka.actor.default-dispatcher-9","level":"INFO","level_value":20000,"HOSTNAME":"8168b4fc1bb6"}
prisma_1  | akka.pattern.AskTimeoutException: Ask timed out on [Actor[akka://single-server/user/$m#2146294253]] after [300000 ms]. Sender[null] sent message of type "com.prisma.deploy.migration.migrator.DeploymentProtocol$Schedule".
prisma_1  | 	at akka.pattern.PromiseActorRef$.$anonfun$defaultOnTimeout$1(AskSupport.scala:595)
prisma_1  | 	at akka.pattern.PromiseActorRef$.$anonfun$apply$1(AskSupport.scala:605)
prisma_1  | 	at akka.actor.Scheduler$$anon$4.run(Scheduler.scala:140)
prisma_1  | 	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:864)
prisma_1  | 	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:109)
prisma_1  | 	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:103)
prisma_1  | 	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:862)
prisma_1  | 	at akka.actor.LightArrayRevolverScheduler$TaskHolder.executeTask(LightArrayRevolverScheduler.scala:328)
prisma_1  | 	at akka.actor.LightArrayRevolverScheduler$$anon$4.executeBucket$1(LightArrayRevolverScheduler.scala:279)
prisma_1  | 	at akka.actor.LightArrayRevolverScheduler$$anon$4.nextTick(LightArrayRevolverScheduler.scala:283)
prisma_1  | 	at akka.actor.LightArrayRevolverScheduler$$anon$4.run(LightArrayRevolverScheduler.scala:235)
prisma_1  | 	at java.lang.Thread.run(Thread.java:748)
prisma_1  | {"key":"error/unhandled","requestId":"local:management:cjh61trxw00030c10wbyen89q","payload":{"exception":"akka.pattern.AskTimeoutException: Ask timed out on [Actor[akka://single-server/user/$m#2146294253]] after [300000 ms]. Sender[null] sent message of type \"com.prisma.deploy.migration.migrator.DeploymentProtocol$Schedule\".","query":"      mutation($name: String!, $stage: String! $types: String! $dryRun: Boolean $secrets: [String!], $subscriptions: [FunctionInput!], $force: Boolean) {\n        deploy(input: {\n          name: $name\n          stage: $stage\n          types: $types\n          dryRun: $dryRun\n          secrets: $secrets\n          subscriptions: $subscriptions\n          force: $force\n        }) {\n          errors {\n            type\n            field\n            description\n          }\n          warnings {\n            type\n            field\n            description\n          }\n          migration {\n            ...MigrationFragment\n          }\n        }\n      }\n      \nfragment MigrationFragment on Migration {\n  revision\n  steps {\n    type\n    __typename\n    ... on CreateEnum {\n      name\n      ce_values: values\n    }\n    ... on CreateField {\n      model\n      name\n      cf_typeName: typeName\n      cf_isRequired: isRequired\n      cf_isList: isList\n      cf_isUnique: unique\n      cf_relation: relation\n      cf_defaultValue: default\n      cf_enum: enum\n    }\n    ... on CreateModel {\n      name\n    }\n    ... on CreateRelation {\n      name\n      leftModel\n      rightModel\n    }\n    ... on DeleteEnum {\n      name\n    }\n    ... on DeleteField {\n      model\n      name\n    }\n    ... on DeleteModel {\n      name\n    }\n    ... on DeleteRelation {\n      name\n    }\n    ... on UpdateEnum {\n      name\n      newName\n      values\n    }\n    ... on UpdateField {\n      model\n      name\n      newName\n      typeName\n      isRequired\n      isList\n      isUnique: unique\n      relation\n      default\n      enum\n    }\n    ... on UpdateModel {\n      name\n      um_newName: newName\n    }\n  }\n}\n\n    ","variables":"{\"name\":\"graphcool\",\"stage\":\"dev\",\"types\":\"type User {\\n  id: ID! @unique\\n  createdAt: DateTime!\\n  updatedAt: DateTime!\\n\\n  email: String @unique\\n  resetPasswordToken: String\\n  resetPasswordExpire: DateTime\\n  password: String\\n  facebookUserId: String @unique\\n  firstName: String\\n  lastName: String\\n}\\n\",\"secrets\":[\"mysecret123\"],\"subscriptions\":[]}","code":"0","stack_trace":"akka.pattern.PromiseActorRef$.$anonfun$defaultOnTimeout$1(AskSupport.scala:595)\\n akka.pattern.PromiseActorRef$.$anonfun$apply$1(AskSupport.scala:605)\\n akka.actor.Scheduler$$anon$4.run(Scheduler.scala:140)\\n scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:864)\\n scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:109)\\n scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:103)\\n scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:862)\\n akka.actor.LightArrayRevolverScheduler$TaskHolder.executeTask(LightArrayRevolverScheduler.scala:328)\\n akka.actor.LightArrayRevolverScheduler$$anon$4.executeBucket$1(LightArrayRevolverScheduler.scala:279)\\n akka.actor.LightArrayRevolverScheduler$$anon$4.nextTick(LightArrayRevolverScheduler.scala:283)\\n akka.actor.LightArrayRevolverScheduler$$anon$4.run(LightArrayRevolverScheduler.scala:235)\\n java.lang.Thread.run(Thread.java:748)","message":"Ask timed out on [Actor[akka://single-server/user/$m#2146294253]] after [300000 ms]. Sender[null] sent message of type \"com.prisma.deploy.migration.migrator.DeploymentProtocol$Schedule\"."}}
prisma_1  | akka.pattern.AskTimeoutException: Ask timed out on [Actor[akka://single-server/user/$m#2146294253]] after [300000 ms]. Sender[null] sent message of type "com.prisma.deploy.migration.migrator.DeploymentProtocol$Schedule".[Bugsnag - local / testing] Error report: com.bugsnag.Report@26417b27
prisma_1  |
prisma_1  | 	at akka.pattern.PromiseActorRef$.$anonfun$defaultOnTimeout$1(AskSupport.scala:595)
prisma_1  | 	at akka.pattern.PromiseActorRef$.$anonfun$apply$1(AskSupport.scala:605)
prisma_1  | 	at akka.actor.Scheduler$$anon$4.run(Scheduler.scala:140)
prisma_1  | 	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:864)
prisma_1  | 	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:109)
prisma_1  | 	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:103)
prisma_1  | 	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:862)
prisma_1  | 	at akka.actor.LightArrayRevolverScheduler$TaskHolder.executeTask(LightArrayRevolverScheduler.scala:328)
prisma_1  | 	at akka.actor.LightArrayRevolverScheduler$$anon$4.executeBucket$1(LightArrayRevolverScheduler.scala:279)
prisma_1  | 	at akka.actor.LightArrayRevolverScheduler$$anon$4.nextTick(LightArrayRevolverScheduler.scala:283)
prisma_1  | 	at akka.actor.LightArrayRevolverScheduler$$anon$4.run(LightArrayRevolverScheduler.scala:235)
prisma_1  | 	at java.lang.Thread.run(Thread.java:748)
```

**Reproduction**
[GitHub repository](https://github.com/Lotafak/PrismaDeployErrorReproduction)

**Expected behavior?**
After running `prisma deploy` database is ready to work.

**Comment by marktani** [comment]
We confirmed that `docker pull prismagraphql/prisma:1.8 && docker-compose up -d` fixed this issue, thanks everyone 🙌

**Comment by willvincent** [comment]
Still an issue in 1.9.0... This is not resolved.

**Comment by Lotafak** [comment]
@willvincent since then I don't have problems with that. Try cleaning everything `docker-compose down -v` and remove running containers and also images `docker rmi $(docker images -q)`, pull the latest prisma version if you haven't already.

**Comment by marktani** [comment]
@willvincent please create a new issue if you are experiencing a similar problem 🙂

**Comment by willvincent** [comment]
Too many problems, we've already moved away from prisma at this point. I want to like it, but too many issues, and we don't have time to wait on eventual resolutions.

**Comment by marktani** [comment]
Hey @willvincent I'm sorry to hear of your troubles. It seems that broadly this issue is resolved already, but maybe you have found a specific case that still shows this error message? If so, sharing this use case with us allows us to timely fix this issue for good 🙂 

Similarly, I would love to hear your feedback for other issues you are facing, so we can improve them.

**Comment by Albert221** [comment]
@Lotafak I just wanted to emphasize, that after a few hours of trying to fix my problem, your solution finally worked and I'll repeat it once again:

>@willvincent since then I don't have problems with that. Try cleaning everything docker-compose down -v and remove running containers and also images docker rmi $(docker images -q), pull the latest prisma version if you haven't already.
